docs/source/markdown/podman-container-clone.1.md.in

```diff
@@ -92,20 +74,7 @@ reservation. So you should always set the value below **--memory**, otherwise th
 hard limit will take precedence. By default, memory reservation will be the same
 as memory limit from the container being cloned.
 
-This option is not supported on cgroups V1 rootless systems.
-
-#### **--memory-swap**=*limit*
-
-A limit value equal to memory plus swap. Must be used with the  **-m**
-(**--memory**) flag. The swap `LIMIT` should always be larger than **-m**
-(**--memory**) value. By default, the swap `LIMIT` will be set to double
-the value of --memory if specified. Otherwise, the container being cloned will be used to derive the swap value.
```

Comment:
You can't refactor this one. It is not equivalent. Please look very closely at the end sentence.

Please, I encourage you to use `hack/markdown-preprocess-review` and look very closely at what it says.